acir_field/src/generic_ark.rs

```diff
@@ -441,6 +485,40 @@ mod tests {
         let max_num_bits_bn254 = crate::generic_ark::FieldElement::<ark_bn254::Fr>::max_num_bits();
         assert_eq!(max_num_bits_bn254, 254)
     }
+
+    #[test]
```

Comment:
what does this print for -1?

Reply:
-1 mod 3 is printed as -1
I added more tests to show more results of the function

Reply:
Thank you, what about 10/3 as mentioned in the issue?

Reply:
You mean I have the number 10 and the number 3. I get the number that will be the result of dividing number 10 by number 3 using the '/' operator. Then I need to take the remainder of the division from the result and output it as 10/3?

Reply:
I added another function for printing `FieldElement` in a pretty format when the division is performed. Firstly I tried to change the `Div` operation to return the result instead of `FieldElement`. This idea changed logic a lot, so I just added a separate function.